test/conftest.py

```diff
@@ -104,9 +104,13 @@ def print_scylla_log_filename(request: pytest.FixtureRequest) -> Generator[None]
 
 
 @pytest.fixture(scope="module")
-async def testpy_testsuite(request: pytest.FixtureRequest, build_mode: str) -> TestSuite:
-    suite_config = find_suite_config(path=request.path)
-    return TestSuite.opt_create(path=str(suite_config.parent), options=request.config.option, mode=build_mode)
+def testpy_suite_config(request: pytest.FixtureRequest) -> Path:
```

Comment:
would like to see documentation for this function as well ( in cql_repl/conftest.py too)